dogfood/client/dogfood_client.go

```diff
@@ -56,7 +58,50 @@ func getCatalogWithTimeout(client pb.ChaosDogfoodClient) ([]*pb.CatalogItem, err
 	return res.Items, nil
 }
 
-// regularly order food for different aniamls
+func printAndLog(logLine string) {
+	fmt.Println(logLine)
+
+	go func() {
```

Comment:
I'm confused, why do we want to write all this random data to /mnt/data/logging whenever we're trying to print to console? They dont seem to need to be so coupled.

Reply:
The point of printAndLog is to give the client Write IO. This is the function that will help us test Disk write disruptions properly

Reply:
I know why we want to have the logging done, but why put it in printAndLog? It seems this could be in gothreads that run separately, I don't see why we kick it off once per print

Reply:
I think this spawned from not knowing what data to write to disk at first. I think you can pull random data from somewhere, but I opted to just use the logs that were already in the code as data to write.